src/components/BitcoinWallet.vue

```diff
@@ -682,15 +682,14 @@ export default {
           this.$store.state.bitcoin.balance.total -
           this.withdraw.amount -
           this.fees[this.withdraw.selectedFee.type].total;
-
-        return remainingBalanceInSats;
+        return parseInt(remainingBalanceInSats);
       } else {
         const remainingBalanceInSats =
           this.$store.state.bitcoin.balance.total -
           this.withdraw.amount -
           (parseInt(this.fees.fast.total) / parseInt(this.fees.fast.perByte)) *
             parseInt(this.withdraw.selectedFee.satPerByte);
-        return remainingBalanceInSats;
+        return parseInt(remainingBalanceInSats);
```

Comment:
```suggestion
        return parseInt(Math.round(remainingBalanceInSats), 10);
```

`parseInt` doesn't actually round it just chops everything off after the decimal. Also you should always supply the radix when using `parseInt` because it can lead to unpredictable behaviour between browsers or with strangely formatted input values.

Reply:
Nice, thanks!